node/service/src/chain_spec.rs

```diff
@@ -532,23 +847,25 @@ pub fn get_from_seed<TPublic: Public>(seed: &str) -> <TPublic::Pair as Pair>::Pu
 		.public()
 }
 
-
 /// Helper function to generate an account ID from seed
-pub fn get_account_id_from_seed<TPublic: Public>(seed: &str) -> AccountId where
-	AccountPublic: From<<TPublic::Pair as Pair>::Public>
+pub fn get_account_id_from_seed<TPublic: Public>(seed: &str) -> AccountId
+where
+	AccountPublic: From<<TPublic::Pair as Pair>::Public>,
 {
 	AccountPublic::from(get_from_seed::<TPublic>(seed)).into_account()
 }
 
 /// Helper function to generate stash, controller and session key from seed
-pub fn get_authority_keys_from_seed(seed: &str) -> (
+pub fn get_authority_keys_from_seed(
+	seed: &str,
+) -> (
 	AccountId,
```

Comment:
Probably omitting formatting fixes from PRs of this size would make things easier

Reply:
That all looks like rustfmt formatting

Reply:
Yes to both, but if we omit formatting for ease of review, then nothing ever gets formatted, which makes things harder over time. I can attempt to revert the formatting to the ad-hoc nature it had before if that seems important, but I'd prefer not to.